cpp/examples/basic/src/process_csv.cpp

```diff
@@ -92,6 +92,9 @@ int main(int argc, char** argv)
   // memory resource.
   rmm::mr::set_current_device_resource(&mr);
 
+  // for testing the CI.
+  i_am_a_dummy_compile_time_error;
```

Comment:
suggestion: avoid ableist name ("dummy"), and use `#error` to make this obvious to readers of the code as well as the compiler.

```suggestion
  #error "Force compilation error to test CI"
```

Reply:
Can also remove the comment if you use this error message.